library/eggdrop

```diff
@@ -0,0 +1,11 @@
+Maintainers: Geo Van O <geo@eggheads.org> (@vanosg)
+
+Tags: 1.6.21, 1.6, stable, latest
+GitRepo: https://github.com/eggheads/eggdrop-docker.git
+GitCommit: def15a99429271388c9f2958dcc1a825f729e3d3
+Directory: 1.6
+
+Tags: 1.8.0, 1.8, develop
+GitRepo: GitRepo: https://github.com/eggheads/eggdrop-docker.git
```

Comment:
There is an extra `GitRepo: ` here. :smile:  Since they are the same git repo, you could just put it once at the top of the file with `Maintainers` and it will apply to both tag lists.